lib/exercises.rb

```diff
@@ -1,19 +1,49 @@
 
 # This method will return an array of arrays.
 # Each subarray will have strings which are anagrams of each other
-# Time Complexity: ?
-# Space Complexity: ?
+# Time Complexity: O(n)
+# Space Complexity: O(n)
 
 def grouped_anagrams(strings)
```

Comment:
👍 However since you're sorting the words, the time complexity is O(n + m log m) where m is the length of the words.  If you assume the words are limited in size, then it would be O(n).